### PR TITLE
Start separation of concerns

### PR DESCRIPTION
**Pull request by JosiasAurel**
It will be good if the utility functions used in the app is separated into chunks that make the code reusable.
- Having better documentation is also essential
- having a testing firebase account will also make sense to have when testing the code for new contributors. When deploying the app, the main firebase config is changed therefore, there is a `localConfig.js` file I am including in development 
locconfig

### CODE REVIEW

**Review by JosiasAurel — COMMENTED** [review]
separation of concerns

**Review by heybereket — CHANGES_REQUESTED** [review]
Good catches/PR, left one piece of feedback.

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+
+// testing the home component
+import Home from "./pages/Home"
```
Comment (heybereket) [inline src/index.js:6]: I don't think this is needed, only the "import App from './App'

**Review by heybereket — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (JosiasAurel) [reply src/index.js]: removed from original file

**Pull request merged**